src/mokka/src/integration-test/java/pl/hycom/mokka/emulator/mock/MockConfigurationControllerIT.java

```diff
@@ -0,0 +1,180 @@
+            .content(requestBody))
+            .andExpect(status().isOk());
+
+        Response response = RestAssured.get("/__admin/mappings");
+        expect().statusCode(200).header(CONTENT_TYPE_KEY, MediaType.APPLICATION_JSON_VALUE).response();
+        String mappings = response.getBody().jsonPath().get("mappings").toString();
+        UUID expectedUUID = UUID.nameUUIDFromBytes(mockConfigurationRepository.findAll().iterator().next().getId().toString().getBytes());
+
+        Assertions.assertTrue(mappings.contains(expectedUUID.toString()));
+        Assertions.assertTrue(mappings.contains(testChange));
+        Assertions.assertEquals(1, StreamSupport.stream(mockConfigurationRepository.findAll().spliterator(), false).count());
+    }
+
+    @Test
+    @Order(3)
+    @WithMockUser(roles="ADMIN")
+    public void request_should_change_flag_MockConfiguration_on_DB_and_remove_WireMockMapping() throws Exception {
+
+        String id = mockConfigurationRepository.findAll().iterator().next().getId().toString();
+
+        mvc.perform(asyncRequest(post("/configuration/" + id + "/disable")).contentType(MediaType.APPLICATION_JSON_VALUE).with(csrf()))
+            .andExpect(status().isOk());
+
+        Response response = RestAssured.get("/__admin/mappings");
+        expect().statusCode(200).header(CONTENT_TYPE_KEY, MediaType.APPLICATION_JSON_VALUE).response();
+        String mappings = response.getBody().jsonPath().get("mappings").toString();
+        String meta = response.getBody().jsonPath().get("meta").toString();
+        UUID expectedUUID = UUID.nameUUIDFromBytes(id.getBytes());
+
+        Assertions.assertFalse(mappings.contains(expectedUUID.toString()));
+        Assertions.assertTrue(meta.contains("0"));
+        Assertions.assertEquals(1, StreamSupport.stream(mockConfigurationRepository.findAll().spliterator(), false).count());
+    }
+
+    @Test
+    @Order(4)
+    public void request_should_change_flag_MockConfiguration_on_DB_and_add_WireMockMapping() throws Exception {
+
+        String id = mockConfigurationRepository.findAll().iterator().next().getId().toString();
+
+        mvc.perform(asyncRequest(post("/configuration/" + id + "/enable")).contentType(MediaType.APPLICATION_JSON_VALUE).with(csrf()))
+            .andExpect(status().isOk());
+
+        Response response = RestAssured.get("/__admin/mappings");
+        expect().statusCode(200).header(CONTENT_TYPE_KEY, MediaType.APPLICATION_JSON_VALUE).response();
+        String mappings = response.getBody().jsonPath().get("mappings").toString();
+        String meta = response.getBody().jsonPath().get("meta").toString();
+        UUID expectedUUID = UUID.nameUUIDFromBytes(id.getBytes());
+
+        Assertions.assertTrue(mappings.contains(expectedUUID.toString()));
+        Assertions.assertTrue(meta.contains("1"));
+        Assertions.assertEquals(1, StreamSupport.stream(mockConfigurationRepository.findAll().spliterator(), false).count());
+    }
+
+    @Test
+    @Order(5)
```

Comment:
To be removed. Test should setup data on its own. Just make a put at the very beginning that will create MockConfiguration.